application/core/services/export_service.py

```
@@ -0,0 +1,178 @@
+        for chapter in chapters:
+            content += f"# {chapter.title}\n\n{chapter.content}\n\n"
+        
+        file_content = content.encode('utf-8')
+        media_type = "application/epub+zip"
+        filename = f"{novel.title.replace(' ', '_')}.epub"
+        
+        return file_content, media_type, filename
+    
+    def _export_to_pdf(self, novel: Novel, chapters: list[Chapter]) -> Tuple[bytes, str, str]:
+        """导出为PDF格式"""
+        # 这里使用临时实现，实际需要使用PDF生成库
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)